core/build.gradle.kts

```diff
@@ -3,12 +3,14 @@ import org.jetbrains.configureBintrayPublication
 plugins {
     `maven-publish`
     id("com.jfrog.bintray")
+    kotlin("plugin.serialization")
 }
 
 dependencies {
     api(project("dependencies", configuration = "shadow"))
 
     implementation(kotlin("reflect"))
+    implementation("org.jetbrains.kotlinx:kotlinx-serialization-runtime:0.20.0-1.4-M3")
     implementation("com.google.code.gson:gson:2.8.5")
```

Comment:
Can't gson be removed now? 🤔

Reply:
No, it's used here: https://github.com/Kotlin/dokka/blob/e1f3417a22942d168ace994734add42cc01d3694/core/src/main/kotlin/plugability/DokkaPlugin.kt#L61 I don't really understand that code so I'd rather not touch it.

Reply:
Okay, I will take care of replacing this with kotlinx-serialization before merging ☺️ 👍

Reply:
Sweet, that'd be great!